ReleaseProcessAutomation/Jira/ServiceFacadeImplementations/JiraProjectVersionFinder.cs

```diff
@@ -48,6 +53,11 @@ public IEnumerable<JiraProjectVersion> FindVersions (string projectKey, string v
         });
   }
 
+  public IEnumerable<JiraProjectVersion> FindVersions (string projectKey)
+  {
+    return FindVersions(projectKey, null);
+  }
+
```

Comment:
We can also remove this overload and use a default value instead. I am fine with both versions, benefit of the default values is that we don't have to introduce another interface method. As you like :)